girder_worker/app.py

```diff
@@ -84,6 +85,11 @@ def gw_task_prerun(task=None, sender=None, task_id=None,
 
         task.job_manager = deserialize_job_info_spec(**jobSpec)
 
+        # Trim off the /job/jobId
+        task.api_url = '/'.join(jobSpec['url'].split('/')[:-2])
```

Comment:
@danlamanna you should now be able to use the '[apiUrl](https://github.com/girder/girder/blob/master/plugins/worker/server/__init__.py#L85)' attribute in the jobSpec instead of splitting this out